static/src/javascripts/boot.js

```diff
@@ -10,8 +10,9 @@ import { markTime } from 'lib/user-timing';
 import { captureOphanInfo } from 'lib/capture-ophan-info';
 import reportError from 'lib/report-error';
 import { cmp } from '@guardian/consent-management-platform';
-import { isInUsa } from 'projects/common/modules/commercial/geo-utils.js';
 import { getCookie } from 'lib/cookies';
+import { isInAustralia } from 'common/modules/commercial/geo-utils';
```

Comment:
we sometimes consider New Zealand as well when doing regional stuff in AUS, wondering if its the same for CMP?

Reply:
Worth checking with stakeholders!

Reply:
i _think_ this is Australia-only